image-rs/tests/signature_verification.rs

```diff
@@ -93,13 +118,53 @@ const SIGSTORE_CONFIG_URI: &str = "kbs:///default/sigstore-config/test";
 #[tokio::test]
 #[serial]
 async fn signature_verification() {
-    common::prepare_test().await;
+    do_signature_verification_tests(&_TESTS, "aa-offline_fs_kbc-resources.json", &None).await;
```

Comment:
I think it would be better to declare this as a `const`, or we add the parameter to `#[case]` macro. How do you like?